server/services/draftsService.ts

```diff
@@ -0,0 +1,74 @@
+import { Request } from 'express'
+import uuid from 'uuid'
+
+export interface Draft<Data> {
+  id: string
+  type: string
+  createdAt: Date
+  updatedAt: Date
+  data: Data
+}
+
+export interface DraftDTO {
+  id: string
+  type: string
+  createdAt: string
+  updatedAt: string
+  data: unknown
+}
+
+export interface Clock {
+  now: () => Date
+}
+
+export default class DraftsService {
+  constructor(private readonly clock: Clock) {}
+
+  private sessionKey(id: string): string {
```

Comment:
does the UUID package provider a UUID type? could we use it here?

Reply:
and else where in the below method signatures

Reply:
Doesn't look like it, just seems to be `string`.